WalletWasabi/JsonConverters/DateTimeOffsetUnixSecondsConverter.cs

```diff
@@ -18,16 +18,14 @@ public override bool CanConvert(Type objectType)
 		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
 		{
 			var stringValue = reader.Value as string;
-			long value;
 			if (string.IsNullOrWhiteSpace(stringValue))
 			{
-				value = default(DateTimeOffset).ToUnixTimeSeconds();
+				return default(DateTimeOffset);
```

Comment:
This is incorrect. This converter must always return a `long` because `epoch` time is in seconds (that's why this class is called XXXXUnixSecondsConverter). In this line you are returning a `DateTimeOffset` while in the `else` a `long` is returned. The compiler doesn't cry because it expects an `object`.

Reply:
Oops! I misunderstood the change, sorry.